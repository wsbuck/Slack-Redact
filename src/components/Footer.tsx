import React from 'react';

import { makeStyles } from '@material-ui/core/styles';
import Paper from '@material-ui/core/Card';
import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';

import hfla from '../assets/hfla.png';
import bid from '../assets/bid.png';

const useStyles = makeStyles(theme => ({
  root: {
    margin: 'auto',
    width: 600,
    padding: theme.spacing(2),
  },
  imageContainer: {
    padding: theme.spacing(2),
    justifyContent: 'center',
    alignContent: 'center',
  },
  image: {
    width: 120,
    maxWidth: '100%',
  },
}));

const Footer = () => {
  const classes = useStyles();

  return (
    <Paper className={classes.root}>
      <Grid container justify='center' spacing={2}>
        <Grid className={classes.imageContainer} container item sm={6}>
          <img className={classes.image} src={hfla} alt="Hack for LA" />
        </Grid>
        <Grid className={classes.imageContainer} container item sm={6}>
          <img className={classes.image} src={bid} alt="Business Improvement Development" />
        </Grid>
        <Grid item xs={12}>
          <Typography variant='caption'>
            created by <a href="https://williambuck.dev">William Buck</a> 
            &nbsp;and <a href="https://www.linkedin.com/in/bonnieawolfe/">Bonnie Wolfe</a>
          </Typography>
        </Grid>
      </Grid>
    </Paper>
  );
}

export default Footer;
