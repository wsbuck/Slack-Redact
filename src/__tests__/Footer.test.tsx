
import React from "react";
import renderer from "react-test-renderer";
import Footer from "../components/Footer";

test("Footer Snapshot", () => {
  const component = renderer.create(
    <Footer />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});