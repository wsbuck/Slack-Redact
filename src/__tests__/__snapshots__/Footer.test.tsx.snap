// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-root-1 MuiPaper-rounded"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root makeStyles-imageContainer-2 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6"
    >
      <img
        alt="Hack for LA"
        className="makeStyles-image-3"
        src="hfla.png"
      />
    </div>
    <div
      className="MuiGrid-root makeStyles-imageContainer-2 MuiGrid-container MuiGrid-item MuiGrid-grid-sm-6"
    >
      <img
        alt="Business Improvement Development"
        className="makeStyles-image-3"
        src="bid.png"
      />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <span
        className="MuiTypography-root MuiTypography-caption"
      >
        created by 
        <a
          href="https://williambuck.dev"
        >
          William Buck
        </a>
        Â and 
        <a
          href="https://www.linkedin.com/in/bonnieawolfe/"
        >
          Bonnie Wolfe
        </a>
      </span>
    </div>
  </div>
</div>
`;
